packages/client/__tests__/pages/ResourceIndex.spec.js

```diff
@@ -59,13 +71,103 @@ describe('Test the resource index page', () => {
         expect(asFragment()).toMatchSnapshot()
     })
     test('should load posts data and display on table', () => {
-        const { debug } = render(
+        render(
             <Router history={history}>
                 <ResourceIndex {...props} />
             </Router>
         )
-
         expect(window.Flamingo.request.get).toHaveBeenCalled()
     })
-    test('should be able to edit a post from the post table', () => {})
+    test('per page button changes the pers page value for the table', () => {})
+    test('can select a row on the resource table', async () => {
```

Comment:
This does not test that a user can select a row. Please find the checkbox, click on it, then find the checkbox again and make sure its checked.